### PR TITLE
Fix warnings generated by purescript-0.8.0

### PR DESCRIPTION
**Pull request by telser**

### CODE REVIEW

File: src/Data/Char/Unicode/Internal.purs [hunk 1]
```diff
@@ -105,17 +105,20 @@ instance showCharBlock :: Show CharBlock where
                          <> " }"
 
 
-gencatZP :: Int
-gencatZP = 67108864
+gencatPF :: Int
```
Comment (telser) [inline src/Data/Char/Unicode/Internal.purs:108]: Not certain why the changes are here for these codes. I generated this file as explained in the README. If it needs to be changed back to the previous order, then something must be different about the unicodedata file right?

Comment (cdepillabout) [inline src/Data/Char/Unicode/Internal.purs:108]: I believe the unicode file gets updated regularly with new code points.  These changes are completely okay.

**Comment by cdepillabout** [comment]
@telser Thanks a lot for this submission!

**Comment by cdepillabout** [comment]
@paf31 I want to release a new version of purescript-unicode, but I'm wondering whether to make it v0.0.2 or v0.1.0 (or v1.0.0?).

This pull request doesn't change anything but the import statements.  So normally it would be v0.0.2.  But it makes it so the package won't compile with anything less than purescript-0.8.  

How does semantic versioning work with relation to compiler versions?

**Comment by garyb** [comment]
We're treating the changes for 0.8-warning-fixing as breaking changes for the library, as all the 0.7.x code is still functional with 0.8, and also we don't want to break code for people on 0.7.x.

The core libraries are going to be bumped to v1.0.0 this time, as we're already treating their versioning as semver (although shifted one part to the right), and in semver 0.x.x basically means "anything goes", so we may as well start using it properly: major bump for breaking change, minor bump for added functionality, patch bump for fixes.

I'd suggest holding off on releasing this for now, as you'll have to bump all the dependencies when we make those changes anyway, @telser already knows this even though he's been fixing libraries like a maniac, so I don't think he'll be offended if you wait to release this too :wink:

**Comment by cdepillabout** [comment]
@garyb Thanks for the explanation!

So when the time comes, this library should also be updated to v1.0.0, right?

**Comment by garyb** [comment]
If you're pretty much happy with the current API and don't intend to change it in the immediate future, yep, I think it makes sense to bump to v1 :+1:

**Comment by cdepillabout** [comment]
Okay, sounds good!
